### DISCUSSION TITLE
Creating new breakpoint dimensions?

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#858] [Q&A] [answered]
My laptop and monitor are both bigger than 1200 px so they fall in the same breakpoint, xl. When I set the width/height of some components it is skewed on my laptop or monitor. I was wondering if there was a way I could establish new breakpoint dimensions?

**Comment by Alex-Wenner-FHR** [comment]
I see setting width to individual breakpoint seems to be the trick maybe?

**Comment by vopani** [comment, marked as answer]
You can set your own breakpoint values in layout.

```python
ui.layout(breakpoint='1335px', zones=[...])
```

**Reply by Alex-Wenner-FHR** [reply]
Thank you!
